packages/kbn-discover-utils/src/components/custom_control_columns/degraded_docs_control.tsx

```diff
@@ -119,3 +129,48 @@ const DegradedDocs = ({
     />
   );
 };
+
+const EnableESQLDegradedDocsControl = ({
+  addIgnoredMetadataToQuery,
+  Control,
+}: {
+  addIgnoredMetadataToQuery: DegradedDocsControlProps['addIgnoredMetadataToQuery'];
+  Control: RowControlComponent;
+}) => {
+  const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
+
+  return (
+    <EuiPopover
+      anchorPosition="upCenter"
+      button={
+        <Control
+          css={{ color: euiThemeVars.euiColorDisabledText }}
+          data-test-subj="docTableDegradedDocDisabled"
+          iconType="indexClose"
+          label={actionsHeaderAriaLabelDegradedAction}
+          onClick={togglePopover}
+        />
+      }
+      closePopover={closePopover}
+      isOpen={isPopoverOpen}
+      className="unifiedDataTable__rowControl"
```

Comment:
```suggestion
```

Would it be enough to have it only for the button?

Reply:
Removed with [refactor(discover): add tooltip and direct action to degrade docs con…](https://github.com/elastic/kibana/pull/195630/commits/961dbde83f29e98ba13d4e4c3996542161db95bf) as there is not more the popover.